src/app/kbv/controllers/self-assessment-router.js

```diff
@@ -0,0 +1,27 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const taxYearToRange = require("../../../utils/tax-year-to-range");
+
+class SelfAssessmentRouterController extends BaseController {
+  locals(req, res, callback) {
+    super.locals(req, res, (err, locals) => {
+      if (err) {
+        return callback(err, locals);
+      }
+
+      const translationKey = "questionWhatTypeSelfAssessment";
+
+      const { yearRangeStart, yearRangeEnd } = taxYearToRange(
+        req.session.question?.info?.currentTaxYear
+      );
+
+      locals.yearRangeStart = yearRangeStart;
+      locals.yearRangeEnd = yearRangeEnd;
+
+      locals.content = req.translate(`fields.${translationKey}.content`);
```

Comment:
Can't we do the translation in the template itself, the same way the title is?

Reply:
It doesnt seem so, i've tried mimicking the title for content but no luck. Cant seem to see another way of dynamically loading it onto the page

Reply:
A solution I have found in the template which achieves it:

`  {% for item in content %}
    <p>{{ translate(item) }}</p>
  {% endfor %}`